<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="to_node3">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>154</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="to_node5">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>132</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>36</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="to_node12">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>234</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="to_node16">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>308</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node1">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>72</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node2">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>109</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node4">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>95</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node6">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>155</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node7">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>208</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node9">
    <property name="geometry">
     <rect>
      <x>466</x>
      <y>234</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node10">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>270</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node11">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>222</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node13">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>277</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node14">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>300</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node15">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>318</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node17">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>315</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node18">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>323</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node19">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>362</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="to_node20">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>401</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="from_node8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>216</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
